test/streamparse/test_dsl.py

```diff
@@ -1,95 +1,68 @@
 import unittest
 import logging
 
-from pystorm.spout import Spout
-from pystorm.bolt import Bolt
-
-from streamparse.dsl.topology import Topology, Grouping, TopologyError
-from streamparse.dsl.spec import Spec
+from streamparse.dsl.exceptions import TopologyError
+from streamparse.dsl.stream import Grouping
+from streamparse.dsl.topology import Topology
+from streamparse.storm.bolt import Bolt
+from streamparse.storm.spout import Spout
```

Comment:
and, as above, may make sense to just alias `Bolt` and `Spout` in the `streamparse.dsl` namespace, although I suppose that part is debateable. At a minimum, `Bolt` and `Spout` should get aliased up to `streamparse.storm`.